mp/src/game/client/momentum/ui/controls/UserComponent.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 #include "vgui_controls/EditablePanel.h"
+#include <steam/isteammatchmaking.h>
```

Comment:
[SteamFriends](https://partner.steamgames.com/doc/api/ISteamFriends#PersonaStateChange_t) is the API with this callback.
```suggestion
#include <steam/isteamfriends.h>
```